### PR TITLE
Fix one typo on step_3 documentation

### PR DESCRIPTION
**Pull request by brownraming**
The rear bracket was missed in step_3 document.

### CODE REVIEW

File: examples/step-3/doc/intro.dox [hunk 1]
```diff
@@ -63,7 +63,7 @@ we need the following:
 - A finite element that describes the shape functions we want to use on the
   reference cell (which in deal.II is always the unit interval $[0,1]$, the
   unit square $[0,1]^2$ or the unit cube $[0,1]^3$, depending on which space
-  dimension you work in. In step-2, we had already used an object of type
+  dimension you work in.) In step-2, we had already used an object of type
```
Comment (bangerth) [inline examples/step-3/doc/intro.dox]: Ah, almost! I think you want the closing parenthesis *before* the period :-)

**Comment by bangerth** [comment]
I'm not sure who @brownraming is -- can you please find me so I can check one thing about your pull request on your machine?

**Comment by bangerth** [comment]
Do you think you could add an entry to `doc/news/changes/minor` that explains your change?

**Comment by bangerth** [comment]
I *think* we worked on this together, right? The change still needs to be uploaded to github, though. Can you ask one of us about how to do that

That includes the changelog.

**Comment by tamiko** [comment]
We're working on it.

**Comment by bangerth** [comment]
Ah, this got merged without the changelog entry. Would you mind adding one in a separate, follow-up pull request?

**Comment by brownraming** [comment]
Yes. I will pull a follow-up request.

**Pull request merged**